hail/build.gradle

```diff
@@ -1,364 +0,0 @@
-buildscript {
```

Comment:
Incoming:
file removed

Reply:
Yep, we just added a gradle plugin called `bloop`, which speeds up scala builds. Should be fine to just drop it.